tests/_internal/concurrency/test_runtime.py

```diff
@@ -24,6 +25,20 @@ def test_runtime_context_manager():
         pass
 
 
+def test_runtime_with_failure_in_loop_thread_start():
```

Comment:
Previously, this would hang for 60s then throw a `pytest.PytestUnhandledThreadExceptionWarning: Exception in thread RuntimeLoopThread`